xslt.go

```diff
@@ -81,3 +81,8 @@ func NewStylesheet(xsl []byte) (*Stylesheet, error) {
 
 	return &Stylesheet{ptr: cssp}, nil
 }
+
+// InitExslt enables exsl namespace. Call this once at program start.
+func InitExslt() {
```

Comment:
Can you just make this an init function (`func init()`)?  Comment can be deleted.